TypeCobol/Compiler/Scopes/Scope.cs

```diff
@@ -77,16 +77,16 @@ public IEnumerator<T> GetEnumerator()
 
             IEnumerator IEnumerable.GetEnumerator() => GetEnumerator();
 
-        /// <summary>
-        /// One Symbol constructor.
-        /// </summary>
-        /// <param name="symbol"></param>
-        public Entry(T symbol)
+			/// <summary>
+			/// One Symbol constructor.
+			/// </summary>
+			/// <param name="symbol"></param>
+            public Entry(T symbol)
```

Comment:
Inconsistent indent : mix of tabs and spaces

Reply:
See last changes